lib/supplierlandingpage/pieChart.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:math';
+
+import 'package:fl_chart/fl_chart.dart';
+import 'package:buildnow/supplierlandingpage/pieChart.dart';
+import 'package:buildnow/supplierlandingpage/PieChart_Data/pieChart_sectionData.dart';
+import 'package:flutter/material.dart';
+
+class PieChartPage extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() => PieChartPageState();
+}
+
+class PieChartPageState extends State {
+  int touchedIndex = 0;
+
+  @override
+  Widget build(BuildContext context) => Container(
+        child: Row(
+            //mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+            children: [
+              SizedBox(width: 80),
+              Stack(alignment: AlignmentDirectional.center, children: [
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(35, 15, 25, 15),
+                  child: Container(
+                    constraints: BoxConstraints(maxHeight: 25, maxWidth: 25),
+                    //decoration: BoxDecoration(shape: BoxShape.circle),
```

Comment:
Remove the comment if it doesn't describe something useful to understanding the code or it is not intended to be changed to this in future